cpp/src/parquet/statistics.h

```diff
@@ -282,8 +282,8 @@ class TypedStatistics : public Statistics {
 
   /// \brief Batch statistics update with supplied validity bitmap
   virtual void UpdateSpaced(const T* values, const uint8_t* valid_bits,
```

Comment:
Can you add a description of the parameters to the docstring? It is not immediately obvious what `num_spaced_values` is...

Reply:
Done.

Reply:
Thank you!